### PR TITLE
Control flow assignments

### PR DESCRIPTION
**Pull request by jasonmeeks**
while and for loop assignments for control flow unit

### CODE REVIEW

File: unit-3-control-flow/lesson-15-user-input-until-exit/tests.py [hunk 1]
```diff
@@ -0,0 +1 @@
+import unittest
```
Comment (rosiecakes) [inline unit-3-control-flow/lesson-15-user-input-until-exit/tests.py]: You could test for exit tho right? Use mock maybe? http://stackoverflow.com/questions/35851323/pytest-how-to-test-a-function-with-input-call

File: unit-3-control-flow/lesson-14-sum-of-numbers-in-list/tests.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        self.assertEqual(sum_of_numbers_in_list([]), 0)
+
+    def test_list_of_1(self):
+        self.assertEqual(sum_of_numbers_in_list([4]), 4)
+
+    def test_list_of_2(self):
+        self.assertEqual(sum_of_numbers_in_list([2,3]), 5)
+
+    def test_list_of_3(self):
+        self.assertEqual(sum_of_numbers_in_list([2,3,4]), 9)
```
Comment (santiagobasulto) [inline unit-3-control-flow/lesson-14-sum-of-numbers-in-list/tests.py]: need moar spazes between commas. PEP8 stuff.

File: unit-3-control-flow/lesson-15-user-input-until-exit/solutions/user_input_until_exit.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+def user_input_until_exit():
+    while raw_input("Enter a string (type 'exit' to quit): ") != 'exit':
```
Comment (santiagobasulto) [inline unit-3-control-flow/lesson-15-user-input-until-exit/solutions/user_input_until_exit.py]: Breaks on Py3.

File: unit-3-control-flow/lesson-14-sum-of-numbers-in-list/tests.py (hunk 2 above)
Comment (santiagobasulto) [inline unit-3-control-flow/lesson-14-sum-of-numbers-in-list/tests.py]: Yep, maybe.

File: unit-3-control-flow/lesson-18-get-element-number/solutions/get_element_number_for_else.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+def get_element_number(a_list, search_term):
+    count = 0
+    for item in a_list:
+        if item == search_term:
+            break
+        count += 1
+    else:
```
Comment (santiagobasulto) [inline unit-3-control-flow/lesson-18-get-element-number/solutions/get_element_number_for_else.py]: Fancy

**Pull request merged**